src/challengePaths/2021/YouRobot.ts

```diff
@@ -0,0 +1,113 @@
+import {
+  myRobotEnergy,
+  myRobotScraps,
+  round,
+  runChoice,
+  Slot,
+  Stat,
+  visitUrl,
+} from "kolmafia";
+import { have } from "../../lib.js";
+import { get } from "../../property.js";
+import { $path, $skill, $slot, $stat } from "../../template-string.js";
+
+export const path = $path`You, Robot`;
+
+type Upgrade = "robot_shirt" | "robot_energy" | "robot_potions";
+
+export class YouRobot {
```

Comment:
Don't make classes that only have static functions, just export each of the functions. This module can act as the class. I'll add this as an eslint rule if I can find one.

All of these functions should have jsdocs

Reply:
Resolved both of these; good shout.